### PR TITLE
Allow export and import of survey questions

### LINKED ISSUE #24: Allow export and import of survey questions

### PR DESCRIPTION
**Pull request by DoxasticFox**
I re-wrote the logic to seed the database so that I can use it to perform imports in Active Admin. The original logic stored part of the seed data in yaml and part of it in Ruby code, which isn't to my taste; The more stuff that can be written declaratively (i.e. in YAML) and the less stuff that can be written imperatively (i.e. in Ruby), the better! Anyway, that part of the PR was mostly done in e12e74e1116405a99e5716c0a98cc219e03bad22.

Then b38b9dc96f15b91afbba07c176d45472f1f8ffd1 contains logic (following the same pattern) to do the exporting. So (with some exceptions you don't need to worry about), the exported database should be the same as the imported database. This property is captured in the spec test in 4bb621dc95e96b6a5be50f1c3cb3485d0281115d.

I snuck a few changes unrelated to #24 into this PR too. I needed 7557841f1719d296c52979149a766462781e3e02 so that I could work on this PR on my MacBook. And in 77abf9618a4567a57d3dcc55b45dcca027608310, I deleted `not_yet_available` from `config/locales/en.yml` because that same key-value pair occurred twice.

Here's how the new feature looks in Active Admin:

Screenshot_20220914_180209

Clicking the "Download" button in the screencap should give you the same contents as `spec/fixtures/files/full.yml`. (Of course, if you, say, modify a glossary entry, the exported yaml file will be different.)

This PR resolves #24.

### CODE REVIEW

File: app/admin/import_export.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        begin
+          unparsed_yaml = request.params[:file].read
+          parsed_yaml = YAML::load(unparsed_yaml)
+          replace_records(parsed_yaml)
+          redirect_success
+        rescue => e
+          redirect_error e.message
+        end
+      elsif request.params[:submission_type] == "export"
+        send_data fetch_records.to_yaml.to_s, filename: "database.yml"
```
Comment (DoxasticFox) [inline app/admin/import_export.rb:71]:
```suggestion
        send_data fetch_records.to_yaml, filename: "database.yml"
```

File: app/admin/import_export.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          notice: 'Database import succeeded!')
+      end
+
+      if request.params[:submission_type].nil?
+        redirect_error
+      elsif request.params[:submission_type] == "import"
+        begin
+          unparsed_yaml = request.params[:file].read
+          parsed_yaml = YAML::load(unparsed_yaml)
+          replace_records(parsed_yaml)
```
Comment (DoxasticFox) [inline app/admin/import_export.rb:65]: If `replace_records` fails, the database might be only partially imported. Ideally, the partial import would be rolled-back in such a case.

Comment (DoxasticFox) [reply app/admin/import_export.rb:65]: Doesn't look like this is feasible. It'll require a transaction that applies to the entire DB. Rails only allows transactions on objects.

**Review by mathob — APPROVED** [review]
I tested export and import without problem.

**Pull request merged**